client/widgets/test-execution-time/time.directive.js

```diff
@@ -0,0 +1,124 @@
+angular.module('probedock.testExecutionTimeWidget').directive('testExecutionTimeWidget', function() {
+  return {
+    restrict: 'E',
+    controller: 'TestExecutionTimeWidgetCtrl',
+    templateUrl: '/templates/widgets/test-execution-time/time.template.html',
+    scope: {
+      organization: '=',
+      test: '='
+    }
+  };
+}).controller('TestExecutionTimeWidgetCtrl', function(api, $scope, projectVersions) {
+
+  var page = 1;
+
+  $scope.pageSize = 25;
+  $scope.maxResults = 100;
+
+  $scope.params = {};
+  $scope.results = [];
+  $scope.loading = true;
+
+  $scope.chart = {
+    data: [],
+    labels: [],
+    options: {
+      pointHitDetectionRadius: 5,
+      showTooltips: false,
```

Comment:
Why? I would expect to be able to hover the mouse over the chart points to see the date, time and exact execution time.